cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -20,9 +20,9 @@
         'name':   'Jeffrey Sumberg',
         'title':  'Human Capital'
         }, {
-        'titles': ['Chief Information Officer (CIO)', 'Assistant Director'],
+        'titles': ['Chief Information Officer (CIO)', 'Assistant Director<sup>*</sup>'],
         'slug':   'technology-and-innovation',
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/e5717da239d2ddf3eecb20eece06cfb3888553ca